server/app/routes/images_bp.py

```diff
@@ -1,13 +1,17 @@
-import requests
 from flask import Blueprint, jsonify, request, send_file
-from db import get_db, get_grid_fs,get_worker_url
+from app.db import get_db, get_tile_fs, get_map_fs
+
 from werkzeug.local import LocalProxy
 from bson.objectid import ObjectId
 import io
 
+from app.tasks import slice
+from app.tasks import thresholding_otsu
+
 db = LocalProxy(get_db)
-fs = LocalProxy(get_grid_fs)
-worker_url = LocalProxy(get_worker_url)
+tileFs = LocalProxy(get_tile_fs)
+mapFs = LocalProxy(get_map_fs)
```

Comment:
```suggestion
tile_fs = LocalProxy(get_tile_fs)
map_fs = LocalProxy(get_map_fs)
```
Предложение все переменные на сервере именовать в едином стиле.

Reply:
✔️